mailer.js

```diff
@@ -26,6 +27,19 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocalTimeString (timeZone, locale) {
+    // if no timeZone is passed, use 'Etc/UTC'
+    moment.tz.setDefault('Etc/UTC')
+    // if no locale is passed, use 'en'
+    locale = locale || 'en'
+    moment.updateLocale(locale)
+    var time = moment()
+    if (timeZone) {
+      time = time.tz(timeZone)
+    }
+    // return a locale-specific time
+    return time.format('LTS (z) dddd, ll')
```

Comment:
> We are specifying the format that says time must come first, then timezone, then full day of the week, then localized date.

IIRC, I believe this is exactly what @ryanfeeley asked for in the meeting about localization.